combinators/challenge 2/styles.css

```diff
@@ -0,0 +1,29 @@
+p {
+	margin-left: 20px;
+}
+
+section h1 + p {
+	font-style: italic;
+	color: black;
+}
+
+section p + p {
+	color: gray;
+}
+
+section p ~ p {
+	color: gray;
+}
+
```

Comment:
Don't think you need both `section p + p` and `section p ~ p`. I'd take `section p + p` out, because it doesn't select anything that `section p ~ p` hasn't already selected, but verify that this is okay before you do it.

Reply:
sure! I think I forgot to delete it! thanks for that Pamela!

Reply:
Done!